epictrack-web/src/components/AppHelpButton/HelpMenuItem.tsx

```diff
@@ -0,0 +1,6 @@
+import { MenuItem, MenuItemProps } from "@mui/material";
+import { Palette } from "styles/theme";
```

Comment:
The help button moves when you click. Please check if that is already handled in this PR

Reply:
hmm interesting, It's happening in chrome. I use firefox and it's not happening there, I will look into it